accordion-shortcodes.php

```diff
@@ -292,7 +292,7 @@ public function accordion_item_shortcode($atts, $content = null) {
 
 		$ids = $this->get_accordion_id($id);
 
-		$accordion_title = sprintf('<%1$s id="%3$s" class="accordion-title%5$s" role="tab" aria-controls="%4$s" aria-selected="false" aria-expanded="false" tabindex="0" %6$s>%2$s</%1$s>',
+		$accordion_title = sprintf('<%1$s id="%3$s" class="accordion-title%5$s" tabindex="0" %6$s><div role="button" aria-controls="%4$s" aria-expanded="false">%2$s</div></%1$s>',
```

Comment:
I'm tempted to make this an actual `button`, instead of a `div` with `role="button"`. My only concern is backwards compatibility with existing installs styling.

Reply:
Thanks for your quick response @philbuchanan! I really appreciate it. I'm tempted to make this a real-life `button` as well. I worked on a similar accordion issue a while back, though, and this was the exact concern I had. Buttons do have their own native styles, which could introduce native but inadvertent styles that might conflict with a site's theme or styling. 

Since the plugin does have the keyboard events necessary for buttons, to maintain the customization feature of this plugin means it should probably be an element without native styles like a `div`. The only thing missing is the conveyed role of button, which is why I wrote it this way. It would be great if this was a full-fledged `button`, but I think some "resetting" of the button's native styles would be required.

Reply:
I agree. I think the ship has sailed on using a `button`. If only I had made better decisions when I first made the plugin back in 2013. ☹️ We live and we learn.